src/lib/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestEditor/schema.js

```diff
@@ -139,6 +139,7 @@ export const schema = () => {
       type: {
         title: 'Type',
         type: 'string',
+        default: invenioConfig.ILL_BORROWING_REQUESTS.defaultType
```

Comment:
sorry, not quite sure how does this work. We set the defaultType to `''` (empty string)?

Reply:
It is maybe overridden in the overlay. To be check what is the value to use when **no default**.

Reply:
As far as I understand, with the current implementation, it will try to use the value used as 'defaultType' in the appropriate configuration file. If it doesn't find the using the value stored in the appropriate configuration file, it will display an empty space in the drop down, almost as if there was no default value code.